pkg/util/mountOpts.go

```diff
@@ -136,7 +137,8 @@ func ProcessOptions(options []string, isTmpfs bool, sourcePath string) ([]string
 	}
 	defaults, err := getDefaultMountOptions(sourcePath)
 	if err != nil {
-		return nil, err
+		logrus.Warnf("Unable to get default mount options: %v", err)
+		logrus.Warnf("If not specified podman will fallback to default [noexec=false, nosuid=true, nodev=true] for mount source: %s", sourcePath)
```

Comment:
nit: Can we get the list of defaults, rather then hard coding them into the warning?

Reply:
Thanks I agree. Getting it from configured defaults.